logs/analyze.go

```diff
@@ -1680,35 +1689,68 @@ func classifyAndSetDetails(logLine state.LogLine, statementLine state.LogLine, d
 				"vacuum_page_dirty": vacuumPageDirty, "read_rate_mb": readRateMb,
 				"write_rate_mb": writeRateMb, "rusage_kernel": rusageKernelMode,
 				"rusage_user": rusageUserMode, "elapsed_secs": rusageElapsed,
-				"pinskipped_pages": pinskippedPages, "frozenskipped_pages": frozenskippedPages,
```

Comment:
?

Reply:
Ah, this is because this is no longer always present, because of the change in 15?

Reply:
Yeah, that's right - this was removed in Postgres 15 and replaced with the number of pages that were scanned in the table.